apps/designer/tailwind.config.cjs

```diff
@@ -0,0 +1,3 @@
+import('tailwindcss').Config;
```

Comment:
why is this `.cjs`?

Reply:
referencee: https://github.com/t3-oss/create-t3-turbo/blob/main/apps/nextjs/tsconfig.json